x/budget/spec/01_concepts.md

```diff
@@ -4,73 +4,61 @@
-    - In Ante Handler
-      - Gas fees to `feeCollectorName` module account
-      - [https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-rc0/x/auth/ante/fee.go#L112-L135](https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-rc0/x/auth/ante/fee.go#L112-L135)
-    - In Budget Module
-      - Part of Atom inflation and gas fees go to different budget pools
-      - Rest stays within `feeCollectorName` module account
-    - In Distribution Module
-      - All amounts in `feeCollectorName` module account go to community fund and staking rewards
-      - [https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-rc0/x/distribution/keeper/allocation.go#L82-L101](https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-rc0/x/distribution/keeper/allocation.go#L82-L101)
-
-- budget module parameter example
-
-  ```json
-  {
-    "budget": {
-      "params": {
-        "epoch_blocks": 1,
-        "budgets": [
-          {
-            "name": "liquidity-farming-20213Q-20221Q",
-            "rate": "0.300000000000000000",
-            "budget_source_address": "cosmos17xpfvakm2amg962yls6f84z3kell8c5lserqta", // Address corresponding to fee_collector module account in cosmoshub case
-            "collection_address": "cosmos10pg34xts7pztyu9n63vsydujjayge7gergyzavl4dhpq36hgmkts880rwl", // 32-bytes address case, sdk.AccAddress(address.Module("farming", []byte("FarmingBudget"))).String()
-            "start_time": "2021-10-01T00:00:00Z",
-            "end_time": "2022-04-01T00:00:00Z"
-          }
-        ]
-      }
-    }
-  }
-  ```
+`x/budget` is a simple Cosmos SDK module that implements budget functionality. It is an independent module from other SDK modules and core functionality is to enable anyone to create a budget plan through parameter change governance proposal. Once it is agreed within the community, voted, and passed, it uses `BudgetSourceAddress` to distribute amount of coins relative to the rate defined in the plan to the `CollectionAddress`. At each `BeginBlock`, collecting all budgets and distribution take place every `EpochBlocks`. `EpochBlocks` is a global parameter that can be modified by a governance proposal.
+
+A primary use case is for Gravity DEX farming plan. A budget module can be used to create a budget plan that has `BudgetSourceAddress` for Cosmos Hub's [FeeCollector](https://github.com/cosmos/cosmos-sdk/blob/master/x/auth/types/keys.go#L15) module account which collects transaction gas fees and part of ATOM inflation. Then, `BudgetSourceAddress` plans to distribute some amount of coins to `CollectionAddress` for farming plan. 
+
+### Budget Plan for ATOM Inflation Use Case
+
+Cosmos SDK's current reward workflow
+
+- In AnteHandler
+
+    - Gas fees are collected in ante handler and they are sent to `FeeCollectorName` module account
+
+    - Reference the following lines of code
+
+      +++ https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-rc0/x/auth/ante/fee.go#L112-L135
```

Comment:
```suggestion
      +++ https://github.com/cosmos/cosmos-sdk/blob/v0.44.0/x/auth/ante/fee.go#L112-L140
```

Reply:
Yes, we should update the version of the Cosmos SDK. Thanks for pointing it out.